TutorialApplication/MainViewController.m

```diff
@@ -69,6 +72,15 @@ - (void)employeesOrderWasRandomized
     [self.tableView reloadData];
 }
 
+- (void)organizationWasChanged:(NSNotification *)notification
+{
+    NSLog(@"%@",[notification.userInfo valueForKey:@"organization"]);
+    NSLog(@"method organizationWasChanged is used");
+    
+    self.organization = [notification.userInfo valueForKey:@"organization"];
```

Comment:
notification.userInfo is a dictionary, why do you access its values by key-value coding?